src/api.raml

```diff
@@ -497,6 +497,13 @@ traits:
         is:
           - paged: { resources: Collections }
           - subjected: { resources: Collections }
+        queryParameters:
+            containing:
+                description: |
+                    Only return collections containing at least one of these items.
+                type: string[]
+                required: false
+                example: [article/1234, interview/5678]
```

Comment:
Using the differentiation "one possible item" (`/article/09560`, recommendations and metrics) vs. "multiple possible items" (`?containing[]=article:09560&containing[]=article:09561`, journal-cms) could be a good solution then.